packages/components/src/components/hds/app-side-nav/base.ts

```diff
@@ -8,9 +8,8 @@ import TemplateOnlyComponent from '@ember/component/template-only';
 export interface HdsAppSideNavBaseSignature {
   Blocks: {
     root?: [];
-    header?: [];
-    body?: [];
-    footer?: [];
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    body?: any;
```

Comment:
I was getting an error complaining about having and empty tuple so I changed back to using `any` types as in the original `SideNav`.

Reply:
This is now no longer happening so I was able to undo this.